api/src/controllers/person.js

```diff
@@ -3,14 +3,18 @@ const ctx = require('../services/data-context');
 const serverUtils = require('../server-utils');
 const auth = require('../auth');
 
-const getPerson = (qualifier) => ctx.bind(Person.v1.get)(qualifier);
+const getPerson = ({ with_lineage }) => ctx.bind(
+  with_lineage
+    ? Person.v1.getWithLineage
+    : Person.v1.get
```

Comment:
```suggestion
  with_lineage === "true"
    ? Person.v1.getWithLineage
    : Person.v1.get
```

Looking at the docs, shouldn't we use `Person.v1.getWithLineage` only when `req.query.with_lineage === "true"`?

Reply:
Yeah, it is probably a good call to be more strict here instead of just a nullish check.  Done!